modules/auxiliary/admin/smb/list_directory.rb

```diff
@@ -55,7 +56,7 @@ def as_size( s )
   def run
     print_status("Connecting to the server...")
     begin
-      connect()
+      connect(versions: [1])
```

Comment:
Just for context; Why do we need to change this to versions 1 here?

Reply:
This module uses a method that is only implemented in the Rex SMB client (`find_first`). It also extracts information from the raw Rex response. Forcing to version 1 will ensure it will use Rex SMB client and not RubySMB client.
This module will need to be redesigned to be able to use RubySMB by default (like many others). This will be taken care of, in the next iteration of this project.